process/transaction/shardProcess.go

```diff
@@ -394,7 +395,7 @@ func (txProc *txProcessor) checkIfValidTxToMetaChain(
 	if txProc.flagMetaProtection.IsSet() {
 		// additional check
 		if tx.GasLimit < txProc.economicsFee.ComputeGasLimit(tx)+core.MinMetaTxExtraGasCost {
-			return process.ErrInvalidMetaTransaction
+			return fmt.Errorf("%w: not enough gas", process.ErrInvalidMetaTransaction)
```

Comment:
is this still backwards compatible?

Reply:
Yes it is. As this is under meta protection flag

Reply:
right, slipped that